CMakeLists.txt

```diff
@@ -35,6 +35,10 @@ if (IRR_COMPILE_WITH_CUDA)
 	enable_language(CUDA)
 endif()
 
+#resource embedding
+if(_IRR_RESOURCE_COMPILE_EVEN_IN_DEBUG)
```

Comment:
you need to declare this option with a proper default, also I'd call it `IRR_EMBED_BUILTIN_RESOURCES`

it also needs to be defined in the installed configure header in `include/irr/config` so the C++ code can check the macro `_IRR_EMBED_BUILTIN_RESOURCES` and decide whether to load from embedded `uint8_t`  array or some file in `include/irr/builtin/`